cmd/serverless/dependencies_linux_amd64.txt

```diff
@@ -107,6 +107,7 @@ github.com/DataDog/datadog-agent/comp/logs/agent
 github.com/DataDog/datadog-agent/comp/logs/agent/agentimpl
 github.com/DataDog/datadog-agent/comp/logs/agent/config
 github.com/DataDog/datadog-agent/comp/logs/agent/flare
+github.com/DataDog/datadog-agent/comp/logs/integrations/def
```

Comment:
It is kind of weird that this components gets included in the serverless binary. Any chance we can ensure it does not get included?

Reply:
I think it's because it's a dependency of the logs agent

Reply:
Yep I think so too